ui/src/utils/jupyterhub.ts

```diff
@@ -1,19 +1,9 @@
 import { JhApp, JhService, JhServiceFull } from '@src/types/jupyterhub';
 import { JhData } from '@src/types/jupyterhub.ts';
-import { DEFAULT_APP_THUMBNAIL } from './constants';
+import { DEFAULT_APP_THUMBNAIL, DEFAULT_PINNED_SERVICES } from './constants';
 
 export const getJhData = (): JhData => {
-  return window.jhdata
-    ? window.jhdata
-    : JSON.parse(localStorage.getItem('jhdata') || '{}');
-};
-
-/**
- * Store JupyterHub data in local storage for non-hub pages
- * @param data
- */
-export const storeJhData = (data: JhData) => {
-  localStorage.setItem('jhdata', JSON.stringify(data));
```

Comment:
Realized we had an endpoint for getting user data, so removed this and replaced with accessing from endpoint and storing in global storage